src/cloud-api-adaptor/test/provisioner/ibmcloud/provision_initializer.go

```diff
@@ -56,8 +56,9 @@ type IBMCloudProperties struct {
 	WorkerCount   int
 	IsSelfManaged bool
 
-	VPC        *vpcv1.VpcV1
-	ClusterAPI containerv2.Clusters
+	VPC                   *vpcv1.VpcV1
+	ClusterAPI            containerv2.Clusters
+	ConfidentialComputing bool
```

Comment:
I can't see in the test code where this is being used? What was the reason for adding this logic now?

Reply:
I wanted to have an easy way to turn on/off the cc while we're working on the code. Is there another better approach I should be using?

Reply:
So this is in the e2e test code space, so if you are planning to add some e2e tests it's needed, but I don't think you are using this anywhere in any tests?

Reply:
The code's gone now so I can't double check, but my point was that this new `ConfidentialComputing` field you are reading from the properties file didn't seem to be used and wasn't setting the `IBM_PEER_POD_CONFIDENTIAL_VM` or whatever the env you created was as far as I could see, so I'm not sure how it would impact anything. Maybe I'm mis-remembering though

Reply:
I had a line in provision_common.go at line 968 that uses it. It might have been hard to see in the diff because the whole section was updated for formatting.

![image](https://github.com/user-attachments/assets/289dbf21-7a00-42b9-b607-c7cf25b0c9d5)